Doc/using/configure.rst

```diff
@@ -299,7 +299,8 @@ General Options
    Defines the ``Py_GIL_DISABLED`` macro and adds ``"t"`` to
    :data:`sys.abiflags`.
 
-   See :pep:`703` "Making the Global Interpreter Lock Optional in CPython".
+   See :pep:`703` "Making the Global Interpreter Lock Optional in CPython" and
+   :ref:`free-threaded-cpython`.
```

Comment:
Should I remove the reference to the PEP here? Add a reference to the PEP in the "what's new" entry?

Reply:
Yeah, I think that makes sense.